spring-shell-samples/src/main/java/org/springframework/shell/samples/standard/ComponentCommands.java

```diff
@@ -56,6 +56,15 @@ public String stringInput(boolean mask) {
 		return "Got value " + context.getResultValue();
 	}
 
+	@ShellMethod(key = "component string required", value = "String input", group = "Components")
```

Comment:
I'd probably combine this with existing `stringInput` by adding a new flag instead of creating a new command.

Reply:
also hesitated, will do

Reply:
But I will remove the default value, otherwise it doesn't make sense